src/redisHelper.ts

```diff
@@ -0,0 +1,149 @@
+import HawkCatcher from '@hawk.so/nodejs';
+import { createClient, RedisClientType } from 'redis';
+import { Effect, sgr } from './utils/ansi';
+
+/**
+ * Helper class for working with Redis
+ */
+export default class RedisHelper {
+  /**
+   * TTL for lock records in Redis (in seconds)
+   */
+  private static readonly LOCK_TTL = 10;
+
+  /**
+   * Redis client instance
+   */
+  private readonly redisClient!: RedisClientType;
+
+  /**
+   * Constructor
+   * Initializes the Redis client and sets up error handling
+   */
+  constructor() {
+    try {
+      this.redisClient = createClient({ url: process.env.REDIS_URL });
+
+      this.redisClient.on('error', (error) => {
+        console.error('[Redis] Client error:', error);
+        if (error) {
+          HawkCatcher.send(error);
+        }
+      });
+    } catch (error) {
+      console.error('[Redis] Error creating client:', error);
+    }
+  }
+
+  /**
+   * Connect to Redis
+   */
+  public async initialize(): Promise<void> {
+    try {
+      await this.redisClient.connect();
+      console.log('[Redis] Connected successfully');
+    } catch (error) {
+      console.error('[Redis] Connection failed:', error);
+      HawkCatcher.send(error as Error);
+    }
+  }
+
+  /**
+   * Close Redis client
+   */
+  public async close(): Promise<void> {
+    if (this.redisClient.isOpen) {
+      await this.redisClient.quit();
+      console.log('[Redis] Connection closed');
+    }
+  }
+  
+  public async getChartDataFromRedis(
```

Comment:
I think such business logic should not be a part of Redis helper. It should be placed in some   factory, maybe `projectChartFactory`